source/reference/metrics.txt

```diff
@@ -0,0 +1,270 @@
+.. _app-services-metrics:
+
+==============================
+App Services Metrics Reference
+==============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+App Services provides a variety of metrics related to App Services and Device
+Sync. You can access your metrics data with the Admin API's
+:admin-api-endpoint:`getMetrics <tag/metrics/operation/adminGetMetrics>` endpoint.
+
+The content on this page is provided for your information. The exact metrics
+returned may be subject to change.
+
+.. Source derived from https://docs.google.com/spreadsheets/d/1cjQopjPUsEKde6TlqO38B0Ko6n4vTwl4tgJ-LHSbf7Y/edit#gid=1354315462
```

Comment:
should this be deleted?

Reply:
Why?

Reply:
i was told at some point to delete comments, but i have no personal issue w keeping. seems vaguely useful for future maintainers of the docs, assuming that spreadsheet is kept up to date.

Reply:
That was the idea. I usually say to delete things that are "commented out", not "comments".